### ISSUE TITLE
emoji lets my dokuwiki edit toolbar disappear

### ISSUE THREAD

**Issue by fjf2002** [ptbrown/dokuwiki-plugin-emoji#12]
Hi ptbrown,

when I install your emoji plugin via the extensions manager, in edit mode my toolbar disappears. I have tried to deactivate some other plugins to test interference, but the problem persists.

I am using Dokuwiki Greebo release.

Sincerely,
fjf2002

**Comment by gfrein** [comment]
I'm having the same problem. Reinstalling the plugin didn't help.

DokuWiki Release 2018-04-22a "Greebo"
Emoji Plugin Release 2018-10-18

**Comment by stevenmilsom** [comment]
Hi. I see the same problem too.

Same setup as smashgordon42 but I have the Bootstrap3 template installed with one of the Bootswatch.com "flatly" theme enabled.

I'm happy to help provide more detailed info if you need it.

Thanks

**Comment by ptbrown** [comment]
If you can, please try the textcomplete_rollback branch to see if that had an effect. The only changed file is script/jquery.textcomplete.js so you can replace that in your wiki.

**Comment by popindavibe** [comment]
I hit the same issue, completely breaks mobile version since navbar cannot be triggered.
I ended up fixing the missing semi-colons reported by the browser console:
```patch
--- ./plugins/emoji/script/jquery.textcomplete_broken.js	2019-02-26 10:58:33.505972039 +0000
+++ ./plugins/emoji/script/jquery.textcomplete.js	2019-02-26 11:09:13.142666087 +0000
@@ -49,2 +49,2 @@
-        args.shift()
-        completer[strategies].apply(completer, args);
+          args.shift();
+          completer[strategies].apply(completer, args);
@@ -384 +384 @@
-    var idProperty = datum.strategy.idProperty
+    var idProperty = datum.strategy.idProperty;
@@ -1192 +1192 @@
-      var arr = range.text.split(sentinelChar)
+      var arr = range.text.split(sentinelChar);
```

**Comment by popindavibe** [comment]
Also you'll probably need to disable/enable the plugin (took several refresh on my server).

**Comment by oukile** [comment]
Hello ptbrown

I meet the same issue with your plugin (fjf2002, smashgordon42. I don't have bootstrap template)

Your plugins provides broken JavaScript and the editing toolbar disappears
https://www.dokuwiki.org/faq:toolbar

I use Release 2018-04-22b "Greebo"

I have tried your tip, i replace the script/jquery.textcomplete.js with the old branch version.
It doesn't solve the pb.

I have recently upgrated from php 5.6 to php 7.2  if this helps (the plugin was ok with php 5.6)

Sincerely
Fabrice
